### ISSUE TITLE
[REQUEST] Research possible integration with PenPot

### ISSUE THREAD

**Issue by gvlx** [finos/a11y-theme-builder#665]
## Suggestion/Concern
Taking advantage of the Open Source [PenPot ](https://penpot.app/) [project](https://github.com/penpot), some sort of integration would be convenient

## Proposed Solution
Export the created design as a library of assets for PenPot
Going further, those assets could be managed directly by the Theme Builder.
Another venue could be having ATB directly integrated within PenPot.

## Related link
Follow similar issue in https://github.com/penpot/penpot/issues/3705

**Comment by aaronreed708** [comment]
Hi @gvlx!  Is this an issue that you'd like to contribute to (e.g. would you like me to assign it to you), or is this functionality that you'd like us to consider adding to the Theme Builder roadmap?  Would you like to join a community meeting and tell us more about PenPot?

**Comment by gvlx** [comment]
Let me see if can bring people from the [Penpot project](https://penpot.app/) and the [Open Source Design community](https://opensourcedesign.net/) in.

Are your meetings always on "Thursdays at 9:15AM EST."? (from the [FINOS calendar](https://www.finos.org/calendar))?

**Comment by aaronreed708** [comment]
Yes, that is correct.  However, we will not be meeting on 11/23 (due to US Thanksgiving) and will probably cancel some meetings at the end of the year due to many people being on holiday.  But we should still have a couple more before the end of the year.

Otherwise, if we want to do something with introductions and demos and some discussion on what is or is not possible, we can instead organize a stand alone meeting.  Now that I'm thinking about it more, this is probably the better option.
